meta/template.md

```diff
@@ -1,48 +1,52 @@
+# Title
 
-# Spec Name
+- Author Name (GitHub username link)
+- Start Date (YYYY-MM-DD)
+- GitHub Issue (GitHub Issue link)
+- GitHub PR (GitHub PR link)
 
-* Status: In Review / Reviewed / Implementing / Implemented
-* Author(s): [Author Name](https://github.com/NuGet)
-* Issue: [1](https://github.com/NuGet/Home/issues/1) Issue Title
+## Summary
 
-## Problem Background
+<!-- One-paragraph description of the proposal. -->
 
-Present the background and the motivation for this design. 
-Present evidence why this is an important problem to tackle.
+## Motivation 
 
-## Who are the customers
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
 
-Customers affected by this change.
+## Explanation
 
-## Goals
+### Functional explanation
```

Comment:
In the past we've never had a functional and technical separation, but I think it's worth a shot. 
Some NuGet features are almost *completely* technical. Others, not so much. 
This spec covers both.

Reply:
Maybe we should make a note that the explanation itself doesn't have to be both?

Like you can have a proposal purely technical or you can have one purely functional?

Reply:
I don't mind that :)
Feel free to make a judgment call.

Reply:
🚢it is my motto